client/galaxy/scripts/components/admin/DataManager/DataManager.vue

```diff
@@ -6,18 +6,10 @@
         <div v-else-if="dataManagers && !dataManagers.length">
             <Alert variant="primary">
```

Comment:
Minor but could use a regular `b-alert` instead of `Alert` here.

Reply:
@guerler Agreed; I didn't add this one.  I'd support separately making a pass removing the custom thin Alert wrapper and standardizing on normal b-alert, but that's not refactoring I'm doing in this PR.  When I looked at this when it first got added I think in a few places there were responses from the server that we needed to update to use the correct class, but that's what we should do instead of having the extra wrapper.